src/index.html

```diff
@@ -5,8 +5,343 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="is-active"> Apple </a>
```

Comment:
```suggestion
            <a href="#" class="nav_link is-active"> Apple </a>
```

Reply:
don't you need same class here?

Reply:
![image](https://user-images.githubusercontent.com/83066417/120189373-efed1980-c21f-11eb-855c-41c3134a1e1f.png)

Reply:
i have got this error and i dont even know how to do another way